instrumentation/sinatra/test/opentelemetry/instrumentation/sinatra_test.rb

```diff
@@ -142,9 +142,10 @@
         'http.status_code' => 404,
         'http.target' => '/missing_example/not_present'
       )
+      _(exporter.finished_spans.first.events).must_be_nil
```

Comment:
Non-blocking suggesting to make this a little more functional.  

```suggestion
      _(exporter.finished_spans.flat_map(&:events).must_be_empty
```